local/rest_api_gcbm/gcbm.py

```diff
@@ -74,8 +82,8 @@ def generate_config(self, file, json_config_file):
         json_filepath = os.path.join(self.dirpath, json_config_file)
 
         mode = "w+"
-        if os.path.exists(json_filepath):
-            mode = "r+"
+        # if os.path.exists(json_filepath):
+        #     mode = "w+"
 
         # AO: disabling in favour of user defined attributes
         # hard_coded_path = f"hard_coded_values/{json_config_file}"
```

Comment:
@ankitaS11 - if you agree with these changes I'm happy for you to remove them. We should try not to let our discussions accumulate in the code.

Reply:
Yes! Very happy with the changes you made, I'll remove these in my next commit.

Reply:
Fixed! :)